libs/api/prisma/migrations/20240723095847_add_currency_to_memberplan/migration.sql

```diff
@@ -0,0 +1,9 @@
+-- CreateEnum
+CREATE TYPE "Currency" AS ENUM ('CHF', 'EUR');
+
+-- AlterTable
+ALTER TABLE "member.plans" ADD COLUMN "currency" "Currency" NOT NULL DEFAULT 'CHF';
```

Comment:
Setting default but removing it later on. This is so that we can't forget to set it but still have all the old memberPlans  set to CHF.